### PR TITLE
Update TweenMax.globalTimeScale typing

### PR DESCRIPTION
**Pull request by leomeloxp**
As per the code in [gsap's repo](https://github.com/greensock/GreenSock-JS/blob/1.20.5/src/uncompressed/TweenMax.js#L545-L558) the type defition for TweenMax.globalTimeScale needed updating.
<details>

<summary>
Please fill in this template.
</summary>

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present). (got a number of issues that were not caused by my (1 line) change. Will look into them if my PR is to be accepted.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/greensock/GreenSock-JS/blob/1.20.5/src/uncompressed/TweenMax.js#L545-L558
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.
</details>

### CODE REVIEW

File: types/gsap/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,10 @@
-// Type definitions for GSAP 1.19
+// Type definitions for GSAP 1.20.6
 // Project: http://greensock.com/
-// Definitions by: VILIC VANE <https://vilic.github.io/>, Robert S <https://github.com/codebelt>, Richard Fox <https://github.com/ProbablePrime>, Philip Bulley <https://github.com/philipbulley>
+// Definitions by:  VILIC VANE <https://vilic.github.io/>,
+//                  Robert S <https://github.com/codebelt>,
```
Comment (ProbablePrime) [inline types/gsap/index.d.ts]: Is this format ok?

The bot uses these to tag reviewers... LIKE ME :)

**Review by ProbablePrime — APPROVED** [review]

File: types/gsap/index.d.ts (hunk 1 above)
Comment (leomeloxp) [reply types/gsap/index.d.ts]: I followed what is being shown in [the readme](https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package).

File: types/gsap/Animation.d.ts [hunk 2]
```diff
@@ -1,89 +1,103 @@
 declare namespace gsap {
-    export class Animation {
-        /** Base class for all TweenLite, TweenMax, TimelineLite, and TimelineMax classes, providing core methods/properties/() => voidality, but there is no reason to create an instance of this class directly. */
+    class Animation {
+        /* Base class for all TweenLite, TweenMax, TimelineLite, and TimelineMax classes, providing core methods/properties/() => voidality,
```
Comment (sheetalkamat) [inline types/gsap/Animation.d.ts]: All these comments are normal comments and earlier they were jsdoc comments. Please change them back to jsDoc comments.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/gsap/index.d.ts (hunk 1 above)
Comment (ProbablePrime) [reply types/gsap/index.d.ts]: Ahh ok

File: types/gsap/Animation.d.ts (hunk 2 above)
Comment (leomeloxp) [reply types/gsap/Animation.d.ts]: @sheetalkamat All JSDoc comments were re-added and correctly formatted.

Comment (leomeloxp) [reply types/gsap/Animation.d.ts]: @sheetalkamat All JSDoc comments were re-added and correctly formatted.

Comment (leomeloxp) [reply types/gsap/Animation.d.ts]: @sheetalkamat All JSDoc comments were re-added and correctly formatted.

**Comment by leomeloxp** [comment]
Hi all. As you can see on my most recent commits all linter issues are now fixed. 

In case it's not 100% clear, these linter errors were not caused by my 1 line change in a function signature, but rather by a previous PR/code change. Please don't let this happen again in the future. DefinitelyTyped in an extremely important repo for everyone using Typescript and having people that are trying to make the library better cleanup after previous PR's that got accepted irresponsibly is really off putting and could give potential contributors PR fatigue or make them simply not want to submit PRs.

**Comment by leomeloxp** [comment]
Hi all. As you can see on my most recent commits all linter issues are now fixed. 

In case it's not 100% clear, these linter errors were not caused by my 1 line change in a function signature, but rather by a previous PR/code change. Please don't let this happen again in the future. DefinitelyTyped in an extremely important repo for everyone using Typescript and having people that are trying to make the library better cleanup after previous PR's that got accepted irresponsibly is really off putting and could give potential contributors PR fatigue or make them simply not want to submit PRs.

**Comment by leomeloxp** [comment]
Hi all. As you can see on my most recent commits all linter issues are now fixed. 

In case it's not 100% clear, these linter errors were not caused by my 1 line change in a function signature, but rather by a previous PR/code change. Please don't let this happen again in the future. DefinitelyTyped in an extremely important repo for everyone using Typescript and having people that are trying to make the library better cleanup after previous PR's that got accepted irresponsibly is really off putting and could give potential contributors PR fatigue or make them simply not want to submit PRs.

**Review by sheetalkamat — APPROVED** [review]

**Comment by sheetalkamat** [comment]
@typescript-bot test this

**Pull request merged**